src/components/Card/Card.tsx

```diff
@@ -90,9 +90,9 @@ function Card({
 
   return (
     <Link
-      to={url ?? ''}
+      to={url}
       className={cardClassName}
-      onClick={onClick}
+      onClick={disabled ? (e) => e.preventDefault() : onClick}
```

Comment:
Is there anywhere still using the onClick prop or can we get rid of it?

Reply:
https://github.com/jwplayer/ott-web-app/blob/develop/src/components/VideoLayout/VideoLayout.tsx#L126C36-L126C36

Reply:
OK, and that's the one we need to dig into series logic for, right? Can you just add a TODO to remove the onClick prop once we don't need it anymore?

Reply:
Not really, it's for displaying the related videos. I'll check if this can be removed